server.go

```diff
@@ -115,7 +118,23 @@ func (s *server) initNoteServiceClient() {
 	s.noteService = noteService
 }
 
+func (s *server) InitAuthGoogleService() {
+	const googleAppId = "871625340195-kf7c2u88u9aivgdru776a36hgel0kjja.apps.googleusercontent.com"
+	const googleRedirectUri = "https://localhost:3000/authenticate/google"
+	var googleSecret string = *googleAuthSecret
```

Comment:
we should check if ```googleAuthSecret``` actualy contain a value and if it's empty display an error message

Reply:
thanks, it's done